src/tool_help.c

```diff
@@ -330,6 +330,8 @@ static const struct helptxt helptext[] = {
    "TLS 1.3 proxy cipher suites"},
   {"    --proxy-tlsauthtype <type>",
    "TLS authentication type for HTTPS proxy"},
+  {"    --tls-use-session-tickets",
+   "Enable TLS session tickets with NSS"},
```

Comment:
The (perhaps not terribly strong) precedent in this file is to put the affected library within parenthesis, so: `"Enable TLS session tickets (NSS)"`